publish.ps1

```diff
@@ -0,0 +1,46 @@
+# **************************************************************************
+#  Copyright (c) Cloud Native Foundation.
+#  SPDX-License-Identifier: Apache-2.0
+# **************************************************************************
+
+<#
+    .SYNOPSIS
+    Publish the CloudEvents.Sdk module to PSGallery
+
+    .PARAMETER NuGetApiKey
+    PowerShell Gallery API Key to be used to publish the module
+#>
+
+param(
+    [Parameter(Mandatory = $true)]
+    [string]
+    $NuGetApiKey
+)
+
+$moduleName = 'CloudEvents.Sdk'
+$OutputDir = Join-Path $PSScriptRoot $moduleName
+
+if (-not (Test-Path $OutputDir)) {
+    New-Item -ItemType Directory -Path $OutputDir | Out-Null
+}
+
+# Build and Test before publish
+$buildScript = Join-Path $PSScriptRoot 'build.ps1'
+$failedTests = & $buildScript -OutputDir $OutputDir
+if ($failedTests -eq 0) {
```

Comment:
See comment above, this becomes obsolete if build/test are separate steps

Reply:
done.